op-supervisor/supervisor/backend/backend.go

```diff
@@ -446,27 +447,32 @@ func (su *SupervisorBackend) CheckMessage(identifier types.Identifier, payloadHa
 	if err != nil {
 		return types.Invalid, fmt.Errorf("failed to check log: %w", err)
 	}
+	if identifier.Timestamp+su.chainDBs.MessageExpiryWindow() < executingDescriptor.Timestamp {
+		su.logger.Debug("Message expired", "identifier", identifier, "payloadHash", payloadHash, "executingTimestamp", executingDescriptor.Timestamp)
+		return types.Invalid, nil
+	}
```

Comment:
Shouldn't we also check that if `identifier.Timestamp > executingDescriptor.Timestamp`, the message is invalid?

Reply:
Yup, good note.